submodules.json

```diff
@@ -1,17 +1,28 @@
 [
-    {
-        "name" : "react-boilerplate",
-        "url" : "https://github.com/react-boilerplate/react-boilerplate",
-        "tags" : ["react", "reactjs"]
-    },
-    {
-        "name" : "node-boilerplate",
-        "url" : "https://github.com/hagopj13/node-express-boilerplate",
-        "tags" : ["node", "nodejs"]
-    }
-    {
-        "name": "repo-boilerplate",
-        "url": "https://github.com/mintel/repo-boilerplate",
-        "tags" : ["none"]
-    }
+
+    "name": "react-boilerplate",
+    "url": "https://github.com/react-boilerplate/react-boilerplate",
+    "tags": ["react", "reactjs"]
+  },
+  {
+    "name": "node-boilerplate",
+    "url": "https://github.com/hagopj13/node-express-boilerplate",
+    "tags": ["node", "nodejs"]
+  },
+  {
+    "name": "jquery-boilerplate",
+    "url": "https://github.com/jquery-boilerplate/jquery-boilerplate",
+    "tags": ["jquery", "jqueryjs", "js"]
+  },
+  {
+    "name": "android-boilerplate",
+    "url": "https://github.com/ribot/android-boilerplate",
+    "tags": ["android", "mvvm", "rxjava"]
+  },
+  {
+    "name": "repo-boilerplate",
+    "url": "https://github.com/mintel/repo-boilerplate",
+    "tags" : ["none"]
```

Comment:
If you have no tags, the repo won't be displayed in the results. You could add tags like `basic`,`repo` or stuff like that.